bin/dev

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+if gem list --no-installed --exact --silent foreman; then
+  echo "Installing foreman..."
+  gem install foreman
+fi
+
+exec foreman start -f Procfile.dev --env /dev/null "$@"
```

Comment:
why /dev/null ?

Reply:
> why /dev/null ?

Honestly? Because that's the line that was in jsbundling-rails :laughing: Based on the blame, I looks like foreman grabbing the .env file was causing an issue with Kemal so this says "don't load it". It shouldn't make a difference for us because we use Dotenv which loads the .env file on Rails load.